library/pk_internal.h

```diff
@@ -84,11 +84,30 @@ static inline mbedtls_ecp_keypair *mbedtls_pk_ec_rw(const mbedtls_pk_context pk)
 static inline mbedtls_ecp_group_id mbedtls_pk_get_group_id(const mbedtls_pk_context *pk)
 {
     mbedtls_ecp_group_id id;
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_key_attributes_t opaque_attrs = PSA_KEY_ATTRIBUTES_INIT;
```

Comment:
Minor: it seems to me these variables could be made local to the `if OPAQUE` block below.

As a reminder, we're allowed to intermix variable declarations and instructions as C99 permits. Most of the code doesn't do it yet as a lot of our codebase pre-dates wide enough C99 support, but these days we're fine :)